client/src/components/styles/_variables.scss

```diff
@@ -3,4 +3,7 @@ $color-grey-black: #525656;
 $color-grey: #dcdcdc;
 $color-snow: #f4f4f4;
 $color-black: #000;
-$break-small: 420px;
+$break-small: 480px;
+$break-medium: 768px;
+$break-large: 1920px;
```

Comment:
I am afraid it is too big breakpoint. 
ipad has 1023px, some screens can be 1024px, another one 1440px

1920px is a pretty big screens

U combine all of them

Reply:
It is not working on my as I want it to look like when I put 1440, that is why I chose such a big breakpoint